src/vscode-bicep/src/deploy/tree/AzManagementGroupTreeItem.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import {
+  ManagementGroupInfo,
+  ManagementGroupsAPI,
+} from "@azure/arm-managementgroups";
+import { DefaultAzureCredential } from "@azure/identity";
+import { uiUtils } from "@microsoft/vscode-azext-azureutils";
+import {
+  AzExtParentTreeItem,
+  AzExtTreeItem,
+} from "@microsoft/vscode-azext-utils";
+
+import { localize } from "../../utils/localize";
+import { GenericAzExtTreeItem } from "./GenericAzExtTreeItem";
+
+export class AzManagementGroupTreeItem extends AzExtParentTreeItem {
+  public readonly childTypeLabel = localize(
+    "managementGroup",
+    "Management Group"
+  );
+  public contextValue = "managementGroup";
+  public label: string = this.childTypeLabel;
+
+  constructor() {
+    super(undefined);
+  }
+
+  public async loadMoreChildrenImpl(): Promise<AzExtTreeItem[]> {
+    const managementGroupsAPI = new ManagementGroupsAPI(
+      new DefaultAzureCredential()
+    );
+
+    const managementGroupInfos =
+      await managementGroupsAPI.managementGroups.list();
```

Comment:
`await` has no effect here and should be removed since the return type is not a promise.

Reply:
Removed here:
https://github.com/Azure/bicep/commit/32baf9abb1cf7f14b091ac240be5d206702d4d9e

Thanks!